src/libs/ReportUtils.ts

```diff
@@ -81,7 +81,12 @@ import * as ReportActionsUtils from './ReportActionsUtils';
 import StringUtils from './StringUtils';
 import * as TransactionUtils from './TransactionUtils';
 import * as Url from './Url';
+<<<<<<< HEAD
```

Comment:
Looks like we've got the leftovers from an un-finished merge conflict resolution.